week_3/project/week_3.py

```diff
@@ -65,12 +86,19 @@ def week_3_pipeline():
             }
         },
     },
+}
+docker = {
+    **docker_resource_config,
     "ops": {"get_s3_data": {"config": {"s3_key": "prefix/stock_9.csv"}}},
 }
 
 
-def docker_config():
-    pass
+@static_partitioned_config(partition_keys=["1", "2", "3", "4", "5", "6", "7", "8", "9", "10"])
+def docker_config(partition_key: str):
+    return {
```

Comment:
you can also do `docker_resource_config | {"ops": {"get_s3_data": {"config": {"s3_key": f"prefix/stock_{partition_key}.csv"}}}}` if you are using Python 3.9